otherlibs/unix/unix.ml

```diff
@@ -380,6 +315,76 @@ external unlink : string -> unit = "unix_unlink"
 external rename : string -> string -> unit = "unix_rename"
 external link : string -> string -> unit = "unix_link"
 
+external execv : string -> string array -> 'a = "unix_execv"
+external execve : string -> string array -> string array -> 'a = "unix_execve"
+external execvp : string -> string array -> 'a = "unix_execvp"
+external execvpe_c :
+            string -> string array -> string array -> 'a = "unix_execvpe"
+
```

Comment:
This function is a refactoring of the logic in the old `Unix.execvpe_ml` for reusabilty in `Unix.create_process` and its cognates.